src/abstract/OApp.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
@Yashiru You can replace this with just `// SPDX-License-Identifier: MIT` for all new contracts added to the repo